specification/schemas/responses/MessageCreated.yaml

```diff
@@ -35,3 +32,5 @@ properties:
           routingPlan:
             description: The routing plan that you requested the message be sent with.
             $ref: ../types/RoutingPlan.yaml
+        links:
```

Comment:
This appears to be in the wrong place in the document, its not coming through when I generate and preview the docs:

![image](https://github.com/NHSDigital/communications-manager-api/assets/879532/293256bd-f6f8-4955-8fc2-fdad58011a59)

Reply:
missing $ on ref tags